nextcloudappstore/user/forms.py

```diff
@@ -11,19 +11,33 @@
 from django.forms import CharField, EmailField, PasswordInput
 from django.utils.translation import gettext_lazy as _
 
+from .odoo import subscribe_user_to_news
+
 
 class SignupFormRecaptcha(forms.Form):
     """integrate a recaptcha field."""
 
     captcha = CaptchaField()
     first_name = CharField(max_length=30, label=_("First name"))
     last_name = CharField(max_length=30, label=_("Last name"))
+    subscribe_to_news = forms.BooleanField(
+        label=_("I agree to receive app developer news and updates from Nextcloud by email (optional)"),
```

Comment:
Just a small point, but the word "agree" doesn't feel like the right word to use for me as it suggests some kind of contractual agreement that is required when in this case it's actually not. `I would like to receive app developer news and updates from Nextcloud by email (optional)` sounds better, I think.

Reply:
thanks, that makes sense